### PR TITLE
Fix/cbi 2123 update rpc client version

### PR DESCRIPTION
**Pull request by DmitriyMolch**
https://cerenetwork.atlassian.net/browse/CBI-2123

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,11 +9,11 @@ require (
 	github.com/aristanetworks/goarista v0.0.0-20200609010056-95bcf8053598 // indirect
 	github.com/centrifuge/go-substrate-rpc-client/v2 v2.0.1
 	github.com/ethereum/go-ethereum v1.9.25
-	github.com/google/addlicense v0.0.0-20210428195630-6d92264d7170 // indirect
```
Comment (AndreiNavoichyk) [inline go.mod:12]: Why did you remove this?

File: go.mod [hunk 2]
```diff
@@ -9,11 +9,11 @@ require (
...
 	github.com/gorilla/websocket v1.4.2 // indirect
 	github.com/prometheus/client_golang v1.4.1
 	github.com/rs/cors v1.7.0 // indirect
 	github.com/stretchr/testify v1.7.0
 	github.com/urfave/cli/v2 v2.3.0
-	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c // indirect
```
Comment (AndreiNavoichyk) [inline go.mod:18]: Why did you remove this?

**Review by AndreiNavoichyk — COMMENTED** [review]
Check linter

File: go.mod (hunk 1 above)
Comment (DmitriyMolch) [reply go.mod:12]: It was removed automatically by the command rub `go mod tidy`

File: go.mod (hunk 2 above)
Comment (DmitriyMolch) [reply go.mod:18]: It was removed automatically by the command rub `go mod tidy`

**Comment by AndreiNavoichyk** [comment]
> Check linter

@DmitriyMolch what about this?

**Review by AndreiNavoichyk — APPROVED** [review]
